keyboard_handler/CHANGELOG.rst

```diff
@@ -0,0 +1,57 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package keyboard_handler
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+0.0.1 (2021-08-16)
+------------------
+* Merge pull request `#5 <https://github.com/ros-tooling/keyboard_handler/issues/5>`_ from lihui815/sonia-str2code
+  added enum_str_to_key_code
```

Comment:
In practice I prefer to clean these up a bit. The commit history isn't always the most readable. And, if we're doing PRs to update - might as well update the version number in `package.xml` at the same time and we can tag after the fact.

Reply:
ok cleaned up and bumped ver#